docs/cli.md

```diff
@@ -72,34 +74,36 @@ The specific design for onchain tools is still in progress and as a result the i
 
 ---
 
-**`nexus tool unregister --tool-fqn <fqn> --owner-cap <object_id>`**
+**`nexus tool unregister --tool-fqn <fqn> --owner-cap [object_id]`**
```

Comment:
Why this change from `<object_id>` to `[object_id]`?

Reply:
`<>` for required  and `[]` for optional

Reply:
Interesting. I did not remember such convention.